### ISSUE TITLE
Intel wireless 7260 does not autoload iwm and correct firmware

### ISSUE THREAD

**Issue by pkgdemon** [furybsd/furybsd-livecd#4]
![photo_2019-11-03_20-26-25](https://user-images.githubusercontent.com/4109732/68095520-375ec880-fe78-11e9-9611-617bedcb7548.jpg)

**Comment by pkgdemon** [comment]
For reference the PCBSD approach was:

# Load the following kernel modules
```kld_list="libiconv libmchain msdosfs_iconv if_bwn if_bwi runfw if_iwm cuse sem geom_uzip fdescfs linsysfs iwn4965fw iwn1000fw iwn5000fw iwn5150fw iwn6000fw iwn6000g2afw iwn6000g2bfw iwn6050fw acpi_video fuse"```

```
# Check for any new wifi devices to setup
for wnic in `sysctl -b net.wlan.devices 2>/dev/null`
do
  cat /etc/rc.conf 2>/dev/null | grep -q "wlans_${wnic}="
  if [ $? -ne 0 ] ; then
    # We have a wifi device, setup a wlan* entry for it
    grep -q "^wlans_" /etc/rc.conf
      if [ $? -eq 0 ] ; then
        WLANCOUNT=`cat /etc/rc.conf | grep "^wlans_" | wc -l | awk '{print $1}'`
      else
      WLANCOUNT="0"
      WLAN="wlan${WLANCOUNT}"
 
      # Save the wlan interface
      echo "wlans_${wnic}=\"${WLAN}\"" >>/etc/rc.conf
      echo "ifconfig_${WLAN}=\"WPA DHCP\"" >>/etc/rc.conf
      echo "ifconfig_${WLAN}_ipv6=\"inet6 accept_rtadv\"" >> /etc/rc.conf
      service netif restart
    fi
  fi
done
```

**Comment by pkgdemon** [comment]
I tried [dsbdriverd](https://github.com/mrclksr/DSBDriverd) 0.6 from ports today with @dszidi.  This did autoload the correct modules but seemed to load them in the wrong order.  
![photo_2019-11-03_20-33-06](https://user-images.githubusercontent.com/4109732/68095646-24002d00-fe79-11e9-955a-f0f2d0aa6af3.jpg)

As evidenced in the screenshot iwm and it's firmware were loaded.  However the device was not recognized with ```sysctl -b net.wlan.devices```.   We had to ```kldload if_iwm``` and kldload if_iwm again before it would work.

**Comment by GimmeLewd** [comment]
I tried [https://forums.freebsd.org/threads/why-i-dont-have-wifi-on-the-thinkpad-x250-and-freebsd-11-1.66157/](url) , it did load my iwm0 but i face new problem. It say failed to reach wpa_supplicant : wpa_cli (8) ping failed.

**Comment by pkgdemon** [comment]
This patch should fix.  Will have a new image soon with resolution.

https://github.com/mrclksr/DSBDriverd/issues/1

**Comment by dpostolov** [comment]
I can also test Intel Wireless-N 7260BN (if_iwm) installed in Intel NUC DN2820FYKH CPU Intel N2830 Bay Trail-M and (after I plug it [now is unplugged]) Intel Wireless AC-3165 (if_iwm) M.2 2230 will be installed in Intel NUC5PPYH Braswell in next releases of FuryBSD.

**Comment by pkgdemon** [comment]
Finally I was able to reproduce with some equipment in my home lab which happened to have an iwm device.  Certain low level utilities such as kldload, and umount were not working properly in a chroot.  I've made 2 braches were I've taken 2 approaches for the last week to fix the issue.  

Although I think one approach is amazing, and provides full read write it requires 8GB of ram.  Due to having seen, and worked with a large number of people only having 4GB of ram I'm settling with an approach that simply removes the chroot, and makes the live file system behave like FreeBSD's with a few extra directories read write for common driver installation.  Another image will be coming that addresses the issue.  I will post the link here when it is tested.
